### PR TITLE
show common used emojis, when use input a single ':', and add cache control to user avatars

### PR DESCRIPTION
**Pull request by steedos**
emoji_top10

### CODE REVIEW

**Comment by gmsecrieru** [comment]
:+1:

File: client/views/app/messagePopupConfig.coffee [hunk 1]
```diff
@@ -116,6 +116,27 @@ Template.messagePopupConfig.helpers
...
+				            ':smiley:',
+				            ':sunglasses:',
+				            ':wink:',
+				            ':innocent:',
+				            ':flushed:',
+				            ':disappointed:',
+				            ':cry:',
+				            ':heart:',
+				            ':broken_heart:',
+				        ]
```
Comment (rodrigok) [inline client/views/app/messagePopupConfig.coffee]: The indentation looks broken here

Comment (steedos) [inline client/views/app/messagePopupConfig.coffee]: sorry for that. 
so what should i do? fix it and pull again?

Comment (rodrigok) [inline client/views/app/messagePopupConfig.coffee]: Just fix and commit, your PR will update automatically

**Comment by steedos** [comment]
I add cache control to user avatars, depending on the uploadDate field in mongodb. so the browser will not reload the avatars if cached. except the user upload a new avatars.

**Comment by rodrigok** [comment]
LGTM

**Pull request merged**